packages/tracing/src/browser/metrics.ts

```diff
@@ -85,6 +93,21 @@ export class MetricsInstrumentation {
             if (tracingInitMarkStartTime === undefined && entry.name === 'sentry-tracing-init') {
               tracingInitMarkStartTime = startTimestamp;
             }
+
+            // capture web vitals
+
+            if (entry.name === 'first-paint') {
+              logger.log('[Measurements] Adding FP (First Paint)');
```

Comment:
Log messages contribute directly to bundle size.

If you think these are going to be useful for debugging outside of the initial implementation, then let's remove the parenthesized long form for the web vital names for the sake of saving some bytes. Here and in other occurrences.

```suggestion
              logger.log('[Measurements] Adding FP');
```

Or even terser:

```suggestion
              logger.log('[Measurements] FP');
```

We can keep the long form as a comment to aid comprehensibility for folks without those TLAs memorized in their brains.